src/dataDisplay/Table/index.tsx

```diff
@@ -0,0 +1,186 @@
+import React from 'react';
+import TableMui from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableSortLabel from '@material-ui/core/TableSortLabel';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import Paper from '@material-ui/core/Paper';
+import Box from '@material-ui/core/Box';
+import Collapse from '@material-ui/core/Collapse';
+
+//import styled from 'styled-components';
+
+import { FixedIcon } from '../..';
+
+// const TextTHead = styled(Text)`
+//   text-transform: uppercase;
+// `;
+
+// const StyledTableCell = styled(TableCell)`
+//   border-bottom: 2px solid ${({ theme }) => theme.colors.separator} !important;
+// `;
+
+export enum TableAlignment {
+  left = 'left',
+  right = 'right',
+  center = 'center',
+}
+
+export enum TableSortDirection {
```

Comment:
does this component support custom sorting? sometimes you can't just use a default algorithm because cell's values are not primitives (like asset cell in our balances tab)

Reply:
Table component is completely changed in this PR https://github.com/gnosis/safe-react-components/pull/30. Ignore its changes for now